modules/remove_map/functions/RemoveMap/fn_RM_doRemoves.sqf

```diff
@@ -0,0 +1,35 @@
+// AUTHORS: TinfoilHate and StatusRed
+#include "script_component.hpp"
+
+params ["_keepMapClasses", "_keepCompassClasses"];
+
+_keepMapClasses append _keepCompassClasses;
+
+[{time > 1}, {
```

Comment:
always use CBA_missionTime instead of time for the CBA time handling

Reply:
Fixed.